File: storydoc-backend/src/main/java/io/storydoc/server/ui/app/MockUIDTO.java
package io.storydoc.server.ui.app;

import io.storydoc.server.ui.domain.MockUIId;
import io.storydoc.server.ui.domain.ScreenShotCollectionId;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class MockUIDTO {

    private MockUIId id;

    private String name;

    private List<ScreenShotDTO> screenshots;

    private List<ScreenShotCollectionId> associatedCollections;

}

File: storydoc-frontend/src/app/api/models/mock-uidto.ts
/* tslint:disable */
/* eslint-disable */
import { MockUiId } from './mock-ui-id';
import { ScreenShotCollectionId } from './screen-shot-collection-id';
import { ScreenShotDto } from './screen-shot-dto';
export interface MockUidto {
  associatedCollections?: Array<ScreenShotCollectionId>;
  id?: MockUiId;
  name?: string;
  screenshots?: Array<ScreenShotDto>;
}
